db/migrate/20250120151959_add_unique_event_index_on_fees.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddUniqueEventIndexOnFees < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def change
+    add_index :fees, %i[pay_in_advance_event_transaction_id charge_id charge_filter_id], unique: true,
+      where: "created_at > '#{Time.current}'", algorithm: :concurrently
```

Comment:
We have to add `pay_in_advance = 't' and pay_in_advance_event_transaction_id is not null` as well I guess

Reply:
also the migration CI is failing because the timestamp is different, do you know how to handle it?

Reply:
Could we clear the `pay_in_advance_event_transaction_id` for one of the duplicates?

Reply:
We are relying on it for many things in API and PDF templates, and since the fees were pushed to the customers for now we cannot really remove it :/